app/main.py

```diff
@@ -17,11 +17,10 @@ def get_person(cls, name: str):
 
 def create_person_list(people: list) -> list:
     res = []
-    i = 0
     for person_data in people:
         person = Person(person_data["name"], person_data["age"])
         res.append(person)
-    for person_data in people:
+    for i, person_data in enumerate(people):
         try:
```

Comment:
By the way, I failed to notice something else: you don't need `try` ... `except` blocks here. You can use `dict.get(key)` method: it returns None if key doesn't exist.